k8s/prometheus/resources/service-accounts.yaml

```diff
@@ -59,13 +59,13 @@ rules:
   - persistentvolumes
   - namespaces
   - endpoints
-  verbs: ["list", "watch"]
+  verbs: ["list", "watch", "get"]
 - apiGroups: ["extensions"]
   resources:
   - daemonsets
   - deployments
   - replicasets
-  verbs: ["list", "watch"]
+  verbs: ["list", "watch", "get", "update"]
```

Comment:
Is it necessary to add `update` permission for this role?

Reply:
The job of the resizer is to resize the deployment, is it not?
Without the permission to update, it fails to do so:

```
I0509 06:42:56.692635       1 nanny_lib.go:108] Resources are not within the expected limits, updating the deployment. Actual: {Limits:map[] Requests:map[cpu:{i:{value:100 scale:-3} d:{Dec:<nil>} s:100m Format:DecimalSI}]} Expected: {Limits:map[cpu:{i:{value:103000000 scale:-9} d:{Dec:<nil>} s: Format:DecimalSI} memory:{i:{value:0 scale:0} d:{Dec:0xc420235890} s: Format:BinarySI}] Requests:map[cpu:{i:{value:103000000 scale:-9} d:{Dec:<nil>} s: Format:DecimalSI} memory:{i:{value:0 scale:0} d:{Dec:0xc420235890} s: Format:BinarySI}]}
E0509 06:42:56.802033       1 nanny_lib.go:110] deployments.extensions "prometheus-kube-state-metrics" is forbidden: User "system:serviceaccount:default:prometheus-kubestatemetricsserviceaccount-bd07" cannot update resource "deployments" in API group "extensions" in the namespace "default"
```

Is there a better way to enable the resizer, other than giving him this (admittedly broad) permission this way?